include/saber/geometry/utility.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef SABER_GEOMETRY_UTILITY_HPP
+#define SABER_GEOMETRY_UTILITY_HPP
+#pragma once
+
+// saber
+#include "saber/utility.hpp"
+#include "saber/geometry/point.hpp"
+#include "saber/geometry/size.hpp"
+
+namespace saber {
+namespace detail {
+
+//Converts from Point to Size
+template<typename T>
+struct ConvertTraits<geometry::Size<T>, geometry::Point<T>>
+{
+    geometry::Size<T> operator()(const geometry::Point<T>& inValue) const
+    {
+        geometry::Size<T> size = {inValue.X(), inValue.Y()};
+        return size;
+    }
+};
+
+// COnverts from Size to Point
+template<typename T>
+struct ConvertTraits<geometry::Point<T>, geometry::Size<T>>
+{
+    geometry::Point<T> operator()(const geometry::Size<T>& inValue) const
+    {
+        geometry::Point<T> point = {inValue.Width(), inValue.Height()};
```

Comment:
Add a `REVISIT` comment here...
C++17 "assignment" from initializer list vs. using "constructor{}": Does it matter?
You will have to make a release builds and look at the compiler generated instructions to make the determination